src/main/java/com/icthh/xm/uaa/service/TenantRoleService.java

```diff
@@ -156,23 +150,25 @@ public void addRole(@Valid RoleDTO roleDto) {
     @SneakyThrows
     public void updateRole(RoleDTO roleDto) {
         Map<String, Role> roles = getRoles();
-        Role role = roles.get(roleDto.getRoleKey());
-        if (role == null) {
-            throw new BusinessException("Role doesn't exist");
-        }
-        // role updating
-        role.setDescription(roleDto.getDescription());
-        role.setUpdatedBy(xmAuthenticationContextHolder.getContext().getRequiredLogin());
-        role.setUpdatedDate(Instant.now().toString());
-        roles.put(roleDto.getRoleKey(), role);
+        Role roleToUpdate = roles.get(roleDto.getRoleKey());
+
+        if (isNull(roleToUpdate)) throw new BusinessException("Role doesn't exist");
```

Comment:
If must have body

Reply:
In java, it is possible to write if without brackets if you have only a single line in the body. Such code become laconic and thus easier to read and understand )

Reply:
please use {} for if even if there is one line

Reply:
@sergeysenja1992  @amedvedchuk  thanks for comments. Let use the "Objectives" argument in the comments. As you can see I posted arguments about why and how such code may fit in this situation.

Reply:
@anton-contributor excuse me but your "argument" about easy to read seems like pure personal preference

Reply:
Ok, let me clarify.  Update this code into a single line makes it laconic (1 shorter than 3 lines on code). If the code is shorter and self-explanatory it becomes more maintainable as it is easier to understand reading fewer lines (without redundancy).  Java language and many other c-style languages allow you to do so.  @amedvedchuk Please provide conter-arguments why it is better to use {}. How does it help me or you when I read a code with single line invocation.

Reply:
@anton-contributor 
There are good practices described in Oracle/Google code conventions.
https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used
```
Braces are used with if, else, for, do and while statements, even when the body is empty or contains only a single statement.
```
Also, this rule present in the `https://github.com/xm-online/xm-uaa/blob/master/qa/checkstyle.xml` rules for this project.

Reply:
I have a suggestion to run checkstyle on project build for non-generated code in order to make the development process faster in similar to these questions. @ishkurko @amedvedchuk @sergeysenja1992  what do you think about it?

Reply:
I think it is a good idea.